backend/app/assets/javascripts/spree/backend/templates/orders/shipment_tracking.hbs

```diff
@@ -0,0 +1,22 @@
+<th>
+  {{ human_attribute_name "spree/shipment" "tracking" }}
+</th>
+<td colspan="4">
+  {{#if editing}}
+    <input type="text" name="tracking" class="fullwidth" value="{{tracking}}">
```

Comment:
If we would wrap this in a `form` we could use the `submit` event instead.

```html
<form id="tracking_form">
```

Notice that we need to have that `id` to connect the "save" button with this form as it needs to be outside of the form. But there is [this barely known attribute of submit buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-form) called `form`. 

Very handy for our use case!